tests/python/lib/testcase.py

```diff
@@ -255,12 +275,22 @@ def extra_class_teardown(cls):
     def should_use_nocc(cls):
         return nocc_env("NOCC_SERVERS_FILENAME", None) is not None
 
+    @classmethod
+    def should_use_k2(cls):
+        return search_k2_bin() is not None
+
+    @classmethod
+    def k2_server_env(cls):
+        env = {"KPHP_MODE": "k2-server", "KPHP_ENABLE_FULL_PERFORMANCE_ANALYZE": "0",
+               "KPHP_PROFILER": "0", "KPHP_USER_BINARY_PATH": "component.so", "KPHP_FORCE_LINK_RUNTIME": "1"}
```

Comment:
It isn't k2 server env, incorrect naming. It's KPHP specific build flags, I would call it smth like `kphp_env_for_k2_build()`.  Or I'd move this stuff to `kphp_builder.py` file.

Reply:
By the way, I haven't found real k2_server_env where I expected to see thinks like `RUST_LOG` or rpc connection pools